packages/autocomplete-core/src/propGetters.ts

```diff
@@ -228,7 +227,7 @@ export function getPropGetters<TItem>({
           providedProps.inputElement ===
             props.environment.document.activeElement &&
           !store.getState().isOpen &&
-          store.getState().query.length >= props.minLength
+          (props.openOnFocus || store.getState().query.length > 0)
```

Comment:
Actually this condition is already checked in the `onFocus` function. We can remove it.

Reply:
https://github.com/francoischalifour/autocomplete.js/pull/31/commits/82a8b5ae6e884a16007da829a41c4a0da11eefe9

like this, right?

Reply:
Yep